WebHostLib/static/assets/faq/common-terms_en.md

```diff
@@ -0,0 +1,91 @@
+
+## Check
+A check is a common term for when you "check", or pick up, a location. In terms of Archipelago this is usually used for
+when a player goes to a location and sends its item or "checks" the location.
+
+## Item
+Items are what you will receive from the server regardless of what game you are playing. This could be a sword, a stat 
+upgrade, a spell, or any other potential receivable for your game.
+
+## Location
+Locations are where items are placed in your game. Whenever you interact with a location, you or another player will
+then receive an item. A location could be a chest, an enemy drop, a shop purchase, or any other interactable that can
+contain items in your game.
+
+## Slot
+A slot is the player name and number assigned during generation. This is equal to the number of `yaml` setting files
+used to generate, or the number specified in the `host.yaml` in a local generation.
+
+## World
+World in terms of Archipelago can mean multiple things and is used interchangeably in many situations. 
+* For the sake of the backend or programming side, world is usually intended to represent the world package for a game 
+integration. For example this would be the entire `worlds/factorio` directory.
+* During gameplay in multiworlds each world is a "slot" or "player." As Archipelago allows multiple players to connect
+to the same slot those players are now sharing a world and completing it cooperatively. For games with native 
+cooperative play Archipelago usually fully supports this and those players would connect to each other and only one
+of them to the multiworld but they would still be sharing the same world.
+
+
+## RNG
+Acronym for "Random Number Generator." Archipelago uses its own custom Random object with a unique seed per generation,
+or, if running from source, a seed can be supplied and this seed will control all randomization during generation as all
+game worlds will have access to it.
+
+## Seed
+A "seed" is a number used to initialize a pseudorandom number generator. Whenever you generate a new game on Archipelago
+this is a new "seed" as it has unique item placement, and you can create multiple "rooms" on the Archipelago site from a
+single seed.
+
+## Room
+Whenever you generate a seed on the Archipelago website you will be put on a seed page that contains all the seed info
+with a link to the spoiler if one exists and will show how many unique rooms exist per seed. Each room has its own 
+unique identifier that is separate from the seed. The room page is where you can find information to connect to the
+multiworld and download any patches if necessary. If you have a particularly fun or interesting seed, and you want to 
+share it with somebody you can link them to this seed page, where they can generate a new room to play it! For seeds 
+generated with race mode enabled, the seed page will only show rooms created by the unique user so the seed page is perfectly safe to share for racing purposes.
+
+## Logic
```

Comment:
I'd probably add a note that something can be considered "out of logic" if accessing a location requires the use of special knowledge that the randomizer does not take into account when calculating if you can access the location.